_tools/packages.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { walk } from "../fs/walk.ts";
+import { relative } from "../path/mod.ts";
+
+export async function discoverPackages() {
+  const packages = [];
+  for await (const entry of Deno.readDir(".")) {
+    if (
+      entry.isDirectory && !entry.name.startsWith(".") &&
+      !entry.name.startsWith("_") && entry.name !== "coverage"
```

Comment:
Note to self: keep this in mind when working on #4200 as we might be adding an "internal" folder.